master/buildbot/steps/package/rpm/rpmbuild.py

```diff
@@ -49,6 +49,7 @@ def __init__(self,
                  specdir='`pwd`',
                  srcrpmdir='`pwd`',
                  dist='.el6',
+                 define={},
```

Comment:
The default value shouldn't be a mutable object. Default to None and set it to an empty dictionary in case it's None?